seagence/README.md

```diff
@@ -0,0 +1,51 @@
+# Seagence
+
+## Overview
+
+[Seagence][1] is a defect monitoring platform for Java applications. Using ExecutionPath technology (aka GPS Tracker for transactions), Seagence detects unknown defects caused by various issues like swallowed exceptions, multithreading issues, and others including defects that disguise in a 200 success HTTP response code.
+
+With the Seagence integration, Seagence continuously analyzes the data stream to detect defects when they occur including the root cause of the defect eliminating the need for debugging and troubleshooting. When a defect is detected, the integration will send an event to Datadog alerting you when there is a defect. Using the out-of-the-box dashboard, you have visibility into the detected defects and root cause to eliminate debugging and troubleshooting. More details can be found on [SeagenceWeb][2].
```

Comment:
```suggestion
The Seagence agent continuously analyzes the data stream to detect defects when they occur, including the root cause of the defect. When a defect is detected, the integration will send an event to Datadog to alert your team. Using the out-of-the-box dashboard, you have visibility into the detected defects and root causes to decrease debugging and troubleshooting time. More details can be found on [SeagenceWeb][2].
```

Reply:
@anawishnoff  I have reformatted this paragraph a little bit. Accepted few suggestions. For others, please check the explanation below.

1) Retained "Seagence continuously analyzes". The Seagence agent only collects data and sends it to Seagence backend. Continuous analysis is done by Seagence backend.

2) Retained "to eliminate debugging and troubleshooting". I need to stay consistent about what I say on the website and what I say on this integration and what Seagence actually does. With defects and root causes in hand, it indeed eliminates the need for debugging and troubleshooting. Hope this clarifies.